signingscript/docker.d/passwords.yml

```diff
@@ -413,6 +413,12 @@ in:
              {"$eval": "AUTOGRAPH_AUTHENTICODE_SHA2_PASSWORD"},
              ["autograph_authenticode_sha2", "autograph_authenticode_sha2_stub"]
              ]
+          - ["https://autograph-external.prod.autograph.services.mozaws.net",
+             {"$eval": "AUTOGRAPH_AUTHENTICODE_EV_USERNAME"},
+             {"$eval": "AUTOGRAPH_AUTHENTICODE_EV_PASSWORD"},
+             ["autograph_authenticode"],
+             "ev"
```

Comment:
I see the "202005" keyid is used above, but I'm a bit confused about how this works..

From what I can tell, these keyids should end up getting parsed into [this property](https://github.com/mozilla-releng/scriptworker-scripts/blob/ffe4fd9391d8857d17b4840d32acd9d98aa0d994/signingscript/src/signingscript/utils.py#L25). But from the looks of it, that property is only ever accessed in [this one place](https://github.com/mozilla-releng/scriptworker-scripts/blob/ffe4fd9391d8857d17b4840d32acd9d98aa0d994/signingscript/src/signingscript/sign.py#L1020).

But `sign_authenticode_file` doesn't ever call into that function and instead extracs keyids out of the format [over here](https://github.com/mozilla-releng/scriptworker-scripts/blob/ffe4fd9391d8857d17b4840d32acd9d98aa0d994/signingscript/src/signingscript/sign.py#L1397). And I can't seem to find anywhere that is joining a `keyid` to the format!

So how do authenticode keyids work??

Reply:
I traced the signing format backwards, and afaict, it looks like it might just need to come from the [task payload](https://github.com/mozilla-releng/scriptworker-scripts/blob/ffe4fd9391d8857d17b4840d32acd9d98aa0d994/signingscript/src/signingscript/task.py#L224)? However, if that's the case, what's the point of putting the signing format in `passwords.yml` (like we do with the "202005" one)? Maybe that's actually a no-op and when :catlee landed #218 he didn't actually switch anything over... Fwiw there are no references to "202005" in Gecko.